pkg/api/rest/resttest/resttest.go

```diff
@@ -57,6 +57,13 @@ func (t *Tester) ClusterScope() *Tester {
 	return t
 }
 
+func (t *Tester) newContext() api.Context {
+	if t.clusterScope {
+		return api.NewContext()
+	}
+	return api.NewDefaultContext()
```

Comment:
nit: not sure how much it impacts the test, but I tend to prefer that we not use a default context and instead use one that has a test namespace value assigned.